### ISSUE TITLE
npm scoped name in package.json breaks the build command

### ISSUE THREAD

**Issue by BigAB** [stealjs/steal-tools#527]
WIth:
```
"steal": "^1.0.0-rc.3",
"steal-tools": "^1.0.0-rc.1"
```

Clone this gist:
https://gist.github.com/BigAB/768d5393fa26e62eda85e0c3cf0f1675
and run 
```
> npm install
    ... 
>./node_modules/.bin/steal-tools build
    ...
> python -m SimpleHTTPServer
```

<del>
_(mac only)_ or whatever you do to serve local files, and look at the page
This bundle will fail because it's looking for the bundle in the wrong place
</del>

![image](https://cloud.githubusercontent.com/assets/76821/19062745/ea61b10e-89b9-11e6-9f4e-222b7a4ee722.png)
Everything will be fine.

...but if you change the `name` property in the package.json to something like:
`@bigab/bug-demo-for-steal-tools`
which is an example of a [user scope](http://blog.npmjs.org/post/116936804365/solving-npms-hard-problem-naming-packages)

and re-run the build and look at the page again:
It won't work because the bundle is in the wrong place
![image](https://cloud.githubusercontent.com/assets/76821/19062890/08e338ae-89bb-11e6-8e4d-8bb6cd3b94f3.png)

<del>
**but now** it loads both the bundle main and the source main for no good reason, and it also seems to load `babel` which is a 1.7MB file
![babel-and-main-twice](https://cloud.githubusercontent.com/assets/76821/19062916/47ee8cf6-89bb-11e6-9848-56b6b66acfe0.png)

So yeah, sort of two separate problems maybe? Maybe connected?
</del>

**Comment by pYr0x** [comment]
hi adam,
i think you have to put the app-name also into the script tag like:
```
<script src="node_modules/steal/steal.production.js" main="bug-demo-for-steal-tools/main"></script>
```

**Comment by matthewp** [comment]
Yeah, the package name is unfortunately necessary in the production script tag name because of `directories.lib`. We might deprecate directories.lib in the future which would make this unnecessary.

**Comment by BigAB** [comment]
Okay, even with that change then, it seems adding a [username or organization scope](http://blog.npmjs.org/post/116936804365/solving-npms-hard-problem-naming-packages) to the `package.json`s `name` property breaks the steal-tools build.

Should I create a new issue or edit the title and or description of this issue?

**Comment by matthewp** [comment]
Editing this one is fine, probably don't have a test for building with a scope name as the main package.json.

**Comment by BigAB** [comment]
I have edited this issue description and title

**Comment by pYr0x** [comment]
where do you put the scoped name of your app?
in your gist i dont see any scoped package.

**Comment by matthewp** [comment]
npm allows you to have "scoped" package names, so the `name` in your package.json could be `@foo/my-package`

**Comment by BigAB** [comment]
@pYr0x in the description above I say:
> ...but if you change the `name` property in the `package.json` to something like:
`@bigab/bug-demo-for-steal-tools`
which is an example of a [user scope](http://blog.npmjs.org/post/116936804365/solving-npms-hard-problem-naming-packages)

So yeah, it's not in the gist, but in the description

**Comment by matthewp** [comment]
Fixed by #533
